newa/__init__.py

```diff
@@ -129,6 +147,24 @@ class Event(Serializable):
     id: 'ErratumId'
 
 
+@define
+class ErrataTool:
+    """
+    Interface to Errata Tool instance
+
+    Its only purpose is to serve information about an erratum and its builds.
+    """
+
+    url : str = "https://errata.devel.redhat.com"
```

Comment:
Is it a good idea to 1. hardcode the URL, 2. use an internal hostname in public code?

Reply:
Good point, I assume there will be some kind of newa config at some point. For now, maybe we can souce url value from env variable?

Reply:
Yep, that sounds like a good way till newa gets a config file.